scripts/update-spicedb.sh

```diff
@@ -1,11 +1,21 @@
 #!/usr/bin/env sh
 
 set -e
+
+VERSION=$(grep spicedb ./spicedb-common/wasm-config.json | sed -E  's/"spicedb":.*"(.*)".*/\1/' | xargs)
```

Comment:
jq is probably a less brittle way to extract this value

Reply:
Yeah but I didn't want to add it as a dependency

Reply:
Jimmy brought up a good point that sed can have portability issues as well. jq is a less brittle and a well used tool so will opt for that instead.